otel-agent/k8s-helm/Chart.yaml

```diff
@@ -1,14 +1,14 @@
 apiVersion: v2
 name: opentelemetry-coralogix
 description: OpenTelemetry agent to which instrumentation libraries export their telemetry data
-version: 0.0.28
+version: 0.0.29
 keywords:
   - OpenTelemetry Collector
   - OpenTelemetry agent
   - Coralogix
 dependencies:
   - name: opentelemetry-collector
-    version: "0.62.1"
+    version: "0.63.0"
```

Comment:
Strictly speaking we could go with this version, which is currently the latest in the fork, but we'd be missing some metrics and improvements. I'd prefer to wait for https://github.com/coralogix/opentelemetry-helm-charts/pull/16 to be resolved and bump afterwards.